test/jdk/java/awt/Robot/HiDPIScreenCapture/HiDPIRobotScreenCaptureTest.java

```diff
@@ -39,14 +38,12 @@
 /**
  * @test
  * @key headful
- * @bug 8073320 8280861
- * @summary  Linux and Windows HiDPI support
+ * @bug 8073320
+ * @summary  Windows HiDPI support
```

Comment:
> The test is run on Linux, so it makes sense to keep Linux in the summary.
True; will rectify that.

> Why do you remove 8280861 from the list of bugs?
While the test technically executes the lines of code changed for 8280861, I cannot imagine a scenario where this test would fail without the 8280861 fix and wouldn't fail with this fix.